CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 1.2 (unreleased)
 ----------------
 
+General
+^^^^^^^
+
+Astropy now requires a Numpy 1.7.0 or later.
```

Comment:
@saimn - can you remove ``a`` (just 'requires Numpy') - when you commit, add ``[ci skip]`` in the commit message since we don't need to re-run the CI